_includes/language_picker.html

```diff
@@ -1,54 +1,52 @@
       {{ site.data.[page.lang].settings.global.language }}
     </span>
   </button>
   {% else %}
   <div class="language-picker__label language-picker__label--text">
-    {% include svg/globe.svg %}
-    <span id="language-picker-{{ include.id }}-description" class="label">
+    <img src="{{ site.baseurl }}/assets/img/globe-blue.svg" alt="" width="12" height="12">
+    <span id="language-picker-{{ include.id }}-description">
       {{ site.data.[page.lang].settings.global.language }}
     </span>
   </div>
   {% endif %}
-
-  <ul id="language-picker-{{ include.id }}" aria-describedby="language-picker-{{ include.id }}-description"
-    class="{% if include.dropdown %}dropdown{% endif %} language-list">
-  {% for locale in site.languages %}
-    <li>
-    {% if page.lang == 'en' %}
-      {% if locale == 'en' %}
-        {% assign locale_url = page.url %}
-      {% elsif locale == page.lang %}
-        {% assign locale_url = page.url %}
-      {% else %}
-        {% assign locale_url = page.url | prepend: locale | prepend: '/' %}
-      {% endif %}
-    {% else %}
-      {% if locale == 'en' and page.lang == 'es' %}
-        {% assign locale_url = page.url | remove_first: 'es/' %}
-      {% elsif locale == 'en' and page.lang == 'fr' %}
-        {% assign locale_url = page.url | remove_first: 'fr/' %}
-      {% elsif locale == page.lang %}
-        {% assign locale_url = page.url %}
-      {% else %}
-        {% assign locale_url = page.url | replace_first: page.lang, locale %}
-      {% endif %}
-    {% endif %}
-      <a
-        href="{{ locale_url | prepend: site.baseurl }}"
-        class="text-no-underline"
-        lang="{{ locale }}"
-      >
-        {{ site.data.[locale].settings.global.locales[locale] }}
-      </a>
-    </li>
-  {% endfor %}
+  <ul
+    id="language-picker-{{ include.id }}"
+    aria-describedby="language-picker-{{ include.id }}-description"
+    class="{% if include.dropdown %}usa-accordion__content{% endif %} language-picker__list"
+  >
+    {% for locale in site.languages %}
+      <li>
+        {% if page.lang == 'en' %}
+          {% if locale == 'en' %}
+            {% assign locale_url = page.url %}
+          {% elsif locale == page.lang %}
+            {% assign locale_url = page.url %}
+          {% else %}
+            {% assign locale_url = page.url | prepend: locale | prepend: '/' %}
+          {% endif %}
+        {% else %}
+          {% if locale == 'en' and page.lang == 'es' %}
+            {% assign locale_url = page.url | remove_first: 'es/' %}
+          {% elsif locale == 'en' and page.lang == 'fr' %}
+            {% assign locale_url = page.url | remove_first: 'fr/' %}
+          {% elsif locale == page.lang %}
+            {% assign locale_url = page.url %}
+          {% else %}
+            {% assign locale_url = page.url | replace_first: page.lang, locale %}
+          {% endif %}
+        {% endif %}
+        <a href="{{ locale_url | prepend: site.baseurl }}" lang="{{ locale }}">
```

Comment:
I see this is old/unchanged, but it's making my brain melt, I feel like we could probably turn this into a custom plugin in ruby and simplify what it's doing (whatever that is....)

(Not for this PR necessarily)

Reply:
Need to do more testing, but I think we might be able to replace this with a one-liner:

```suggestion
        <a
          href="{{ page.url | remove_first: 'es/' | remove_first: 'fr/' | prepend: locale | remove_first: 'en/' | prepend: '/' | prepend: site.baseurl }}"
          lang="{{ locale }}"
        >
```

Reply:
omg winner

Reply:
I think I misunderstood how `remove_first` works, so this might not work exactly as we want all the time, particularly when the path includes `en/`, `fr/`, `es/` somewhere other than the start of the URL.

To your original suggestion, I think we could probably adapt our [existing locale filters](https://github.com/18F/identity-site/blob/main/_plugins/locale_url_filter.rb) to support what we need here. I might do it separately.

Reply:
Follow-up, for posterity: #676